frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -1,23 +1,27 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
-import { sortable } from '@patternfly/react-table';
 import * as classNames from 'classnames';
+import * as plugins from '../plugins';
```

Comment:
Please avoid using `public/plugins.ts` directly if possible.

There's a React hook (`useExtensions`) and HOC (`withExtensions`) available since #3383 - you can use them to consume Console extensions in a React-friendly way.